src/utils/hooks/api/vaults/get-vault-data.ts

```diff
@@ -114,6 +115,8 @@ const getVaultData = async (vault: VaultExt, accountId: AccountId, prices: Price
     getTokenPrice(prices, WRAPPED_TOKEN_SYMBOL)?.usd
   );
 
+  const collateralTokenCondition = getCurrencyEqQuery(vault.backingCollateral.currency)
```

Comment:
Not sure about this function name either :)

Reply:
Same function as in the previous case :)